cypress/integration/persianBodySpec.js

```diff
@@ -50,6 +50,12 @@ describeForLocalAndTest('Article Body Tests', () => {
     const imageHasLoaded = getElement('figure').eq(2);
 
     visibleImageWithCaption(imageHasLoaded);
+
+    // NB: If this test starts failing unexpectedly it's a good sign that the dom is being
```

Comment:
Does this need duplicating in each spec? Could we add it to a readme then just have a link here?

Reply:
I think its nicer here, means its less likely to be ignored and readmes are typically huge and get ignored

Reply:
I think duplicating them is fine. My only other thought was whether we could have a test specifically for the expected hydration behaviour so that will fail along with these and be self-documenting, i.e. `it does not clear the DOM during hydration` or whatever.